docs/chain/collator.md

```diff
@@ -544,8 +554,24 @@ chart={`flowchart TD
     K -->|yes|L("Balance with expired lock")
     K -->|no|J
     L -->|unlock_unstaked|A
-`}
-/>
+    
+    %% style assignement
+    A:::spearmint
+    B:::green
+    I:::yellow
+    G:::yellow
+    H:::rose
+    J:::rose
+    K:::rose
+    L:::red
```

Comment:
Maybe we can write some info regarding the meaning of the colors? Maybe we can also have something like: green reflects that blocks are build, dark green means that the collator can also be selected for the next session, red means no collation but locked coins etc.